CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 -   [ENH] Add `names_expand` and `index_expand` parameters to `pivot_wider` for exposing missing categoricals. Issue #1108 @samukweku
 -   [ENH] Add fix  for slicing error when selecting columns in `pivot_wider`. Issue #1134 @samukweku
 -   [ENH] `dropna` parameter added to `pivot_longer`. Issue #1132 @samukweku
+-   [INF] Update `mkdocstrings` version and to fit its new coming features. PR #1138 @Zeroto521
```

Comment:
Oooh, this is exciting! What new features are coming?

Reply:
Sine 0.19 mkdocstrings have to set a handler. This is a breaking changing.

But good for us, we can link to sphinx doc easily. https://mkdocstrings.github.io/usage/#cross-references-to-other-projects-inventories